.github/workflows/ci-badger-tests.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches:
       - main
       - 'release/v*'
-  pull_request_target:
+  pull_request:
```

Comment:
hey @billprovince I made this fix to your PR. 

There are 2 issues with our CI here:
- `pull_request_target` is what is causing this issue, and this was added per recommendation from coveralls to support coverage
- `cache` cleanup was also done on GitHub Actions (as we suspected an issue there).

cc: @adityasadalage and @joshua-goldstein ... lets do a check on these items tomorrow.

Reply:
Thanks for catching this and diagnosing the problem...  I suggest we merge the CI fix in https://github.com/dgraph-io/badger/pull/1944 first, and then we can merge Bill's PR (and hopefully CI will run as expected on here)